server/routes/qnaRoutes.js

```diff
@@ -5,15 +5,17 @@ const axios = require('axios');
 const config = require('../../config.js');
 
 router.get('/questions', (req, res) => {
-  let id = Number(req.data.id);
+  let id = Number(28212);
 
-  axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=${id}&page=1&count=100`, {
+  //&page=1&count=100
```

Comment:
Do you need this comment? If so, cool!

Reply:
Yeah I do. I don't want to limit in the server request. Can manage response in front rend.